src/System.Runtime.Numerics/src/System/Numerics/BigNumber.cs

```diff
@@ -760,7 +760,6 @@ internal static bool TryFormatBigInteger(BigInteger value, ReadOnlySpan<char> fo
             }
             if (value._sign < 0)
             {
-                string negativeSign = info.NegativeSign;
                 for (int i = info.NegativeSign.Length - 1; i > -1; i--)
                     rgch[--ichDst] = info.NegativeSign[i];
```

Comment:
Would it be better/valueable to replace these two `info.NegativeSign` reads with the local?  Theoretically the IL has to rewalk the property every time in the loop because it could have been changed by another thread.

Seems like this was setting up for that but forgot to finish.

Reply:
We could.  NegativeSign just reads a field and is almost certainly being inlined.  No preference.

Reply:
I think I'd prefer to change it since it's "better" and seems to have been what the author intended.  sharplab.io suggests it saves about 10 instructions... so if a computer starts doing this operation now then around the time the CPU fan stops working it might have been worth the time it took me to determine that :smile:.  (I don't know how much of the 10 was a reduction of states in the register schedule vs the cost of actually doing the dereferencing... I just looked at the last instruction's relative offset before and after)